docs/content/_navigation.json

```diff
@@ -120,23 +120,17 @@
           {
             "title": "Op retries",
             "path": "/concepts/ops-jobs-graphs/op-retries"
-          }
-        ]
-      },
-      {
-        "title": "Graphs",
-        "path": "/concepts/ops-jobs-graphs/graphs",
-        "children": [
+          },
           {
-            "title": "Graphs",
+            "title": "Op Graphs",
```

Comment:
Inconsistent casing in concept hierarchy

![Screenshot 2023-08-09 at 10 48 18 AM](https://github.com/dagster-io/dagster/assets/28738937/7c53a541-fd11-46d2-9f25-cf093a3681a6)

Reply:
👍